docs/content/preview/releases/yba-releases/v2.17.md

```diff
@@ -24,6 +24,288 @@ Included here are the release notes for all releases in the **YugabyteDB Anywher
 
 For an RSS feed of all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
 
+## v2.17.3.0 - April 12, 2023 {#v2.17.3.0}
+
+**Build:** `2.17.3.0-b142`
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.17.3.0-b142
+```
+
+### New Features
+
+* [PLAT-3634] Making logging controller authenticated.
+* [PLAT-3884] Add support for TLS-enabled universes
+* [PLAT-5028] Support separate gflags for read-replicas
+* [PLAT-5310] Add search functionality for universe tables lists
+* [PLAT-5368] [UI] Implement Custom Notification Email Template
+* [PLAT-5370] [UI] Add Custom Variable Support for Email Notification Template
+* [PLAT-5380] Set gFlag to use hostname for onprem TLS enabled universes using DNS
+* [PLAT-5443] Enabling python 3.10 and 3.11 support for ybcloud
+* [PLAT-5712] Account for disks for paused universe
+* [PLAT-5734] Support EAR enabled universes for attach/detach
+* [PLAT-5796] [YBA Enablement] Cloud Provider Config - Onprem(sudo, non-sudo)
+* [PLAT-5888] [PLAT-5889] Adding Re-encryption modules for all KMS providers universe key
+* [PLAT-6559] Add create on prem provider form
```

Comment:
Two 6559s?

Reply:
hmm for some reason , delete duplicates haven’t work well this time. Thanks Ill take a closer look.